src/Core/Models/Config/BotConfiguration.cs

```diff
@@ -127,6 +127,10 @@ public class BotConfiguration
         [Description("The minimum time between being able to use the latex command again, in milliseconds")]
         public int LatexTimeoutMillis { get; set; }
 
+        [YamlMember(Alias = "guruDeleteTresholdMillis")]
+        [Description("The maximum time before a message can be deleted when the format code command is used by a guru, in milliseconds")]
```

Comment:
command is used by Guru and up, and is not a guru specific requirement. Change name so it describes function

Reply:
Changed it to CodeFormatterDeleteTresholdMillis